change/@fluentui-react-radio-f2a00719-e7f8-45c1-a97b-76b8505bf4f4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30246&commitId=bdbf39e31bfb198dab37eea752110c3686edfdd6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Radio Converged</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.disabled - Dark Mode.hover.chromium.png|3| Changed|
|Radio Converged.disabled.active.chromium.png|5| Changed|
|Radio Converged.disabled+checked.default.chromium.png|5| Changed|
|Radio Converged.disabled.hover.chromium.png|5| Changed|
|Radio Converged.disabled - Dark Mode.active.chromium.png|3| Changed|
|Radio Converged.disabled.rest.chromium.png|5| Changed|
|Radio Converged.disabled - Dark Mode.rest.chromium.png|3| Changed|
|Radio Converged.disabled+checked - Dark Mode.default.chromium.png|3| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|16| Changed|
|Radio Converged.group-disabled.default.chromium.png|16| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The screenshot diffs are a result of correcting the disabled border color for the radio indicator. The changed visuals now match the figma ✅